packages/xo-web/src/xo-app/logs/backup-ng-logs.js

```diff
@@ -336,6 +336,25 @@ export default decorate([
     effects: {
       toggleState,
     },
+    computed: {
+      backupLogs: (_, { logs, vms }) =>
+        defined(logs.backup, []).map(log =>
+          log.tasks !== undefined
+            ? {
+                ...log,
+                vmNames: (() => {
+                  let names
+                  log.tasks.forEach(task => {
+                    const vm = vms[task.data.id]
+                    vm !== undefined &&
+                      (names || (names = [])).push(vm.name_label)
+                  })
+                  return names
+                })(),
```

Comment:
```js
vmNames: map(
  pick(vms, map(log.tasks, 'data.id')),
  'name_label'
)
```

Reply:
You're right, even though there shouldn't be that many tasks, right?
What do you think of:
```js
vmNames: map(log.tasks, ({ data }) =>
  get(() => vms[data.id].name_label)
)
```

Reply:
`vmNames` will contains `undefined` entries.

Reply:
Yes but that's fine, right?